src/types/ContactTagGroup.ts

```diff
@@ -0,0 +1,7 @@
+export interface ContactTagGroupRef {
+    id: number;
+    name: string;
+    color: string;
+}
+
+export interface ContactTagGroup extends ContactTagGroupRef {}
```

Comment:
It's empty now but we may expand some props here, while living references as is. Not sure if this separation is necessary ATM? 🤔